localization/french/wc_other_titles_l_french.yml

```diff
@@ -1,453 +1,802 @@
- d_argent_dawn:0 "Aube d'argent"
- d_argent_dawn_adj:0 "argentée"
- d_darkspear:0 "Sombrelances"
- d_darkspear_adj:0 "sombrelance"
+ e_black_empire_adj:0 "noire"
+ e_black_empire_article:0 "l'"
+
+ ### Tuskarrs ###
  d_kalu_ak:0 "Kalu'ak"
  d_kalu_ak_adj:0 "kalu'ak"
+
+ ### Hozen ###
  d_hozu_mozu:0 "Hozu'mozu"
- d_hozu_mozu_adj:0 "hozu'mozu"
- d_frostfeather:0 "Givreplume"
+ d_hozu_mozu_adj:0 "hozu'mozian"
+
+ ### Harpies ###
+ d_frostfeather:0 "Givreplumes"
  d_frostfeather_adj:0 "givreplume"
- d_windfury:0 "Furie-des-vents"
- d_windfury_adj:0 "furie-des-vents"
- d_cenarion_hold:0 "Fort cénarien"
- k_blackrock:0 "Ombreforge"
- k_blackrock_adj:0 "ombreforgienne"
- d_syndicate:0 "Syndicat"
- d_syndicate_adj:0 "syndicale"
- k_dun_niffelem:0 "Dun Niffelem"
- k_dun_niffelem_adj:0 "dunniffelem"
- e_kolkar:0 "Kolkar"
+ d_frostfeather_article:0 "les "
+
+ d_windfury:0 "Ensorçailes"
+ d_windfury_adj:0 "ensorçaile"
+ d_windfury_article:0 "les "
+
+ ### Centaurs ###
+ e_kolkar:0 "Clan Kolkar"
  e_kolkar_adj:0 "kolkar"
- d_woodpaw:0 "Griffebois"
- d_woodpaw_adj:0 "griffebois"
- d_savage_hill:0 "Colline sauvage"
- d_savage_hill_adj:0 "de la Colline sauvage"
- e_green_dragonflight:0 "Vol draconique vert"
- e_green_dragonflight_adj:0 "verdoyante"
- e_black_dragonflight:0 "Vol draconique noir"
- e_black_dragonflight_adj:0 "obsidienne"
- e_red_dragonflight:0 "Vol draconique rouge"
- e_red_dragonflight_adj:0 "vermillonne"
- e_blue_dragonflight:0 "Vol draconique bleu"
- e_blue_dragonflight_adj:0 "azur"
- e_bronze_dragonflight:0 "Vol draconique de bronze"
- e_bronze_dragonflight_adj:0 "bronze"
- d_silverwing_sentinels:0 "Sentinelles d'Aile-argent"
- d_silverwing_sentinels_adj:0 "sentinelle"
- d_cenarion_circle:0 "Cercle cénarien"
- d_cenarion_circle_adj:0 "cénarienne"
- d_wardens:0 "Gardiennes"
- d_wardens_adj:0 "gardienne"
- k_wildhammer:0 "Grim Batol"
- k_wildhammer_adj:0 "grim-batol"
- d_bloodsail_buccaneers:0 "Voile sanglante"
- d_northsea_freebooters:0 "Flibustiers des mers du Nord"
- d_spirestone:0 "Pierre-du-pic"
- d_smoulderthorn:0 "Brûleronce"
+ e_kolkar_article:0 "le "
+
+ ### High Elves ###
+ d_sunfury:0 "Solfuries"
+ d_sunfury_adj:0 "solfurie"
+ d_sunfury_article:0 "les "
+
+ ### Draenei ###
+ k_exodar:0 "Exodar"
+ k_exodar_adj:0 "exodarienne"
+ k_exodar_article:0 "l'"
+
+ ### Nagas ###
+ d_coilfang:0 "Tribu Glisseroc"
+ d_coilfang_adj:0 "glisseroc"
+ d_coilfang_article:0 "the "
+
+ ### Tol'vir ###
+ ### Empty ###
+
+ ### Illidari ###
+ k_illidari:0 "Illidaris"
+ k_illidari_adj:0 "illidari"
+ k_illidari_article:0 "les "
+
+ ### Society Titles ###
  k_cenarion_circle:0 "Cercle cénarien"
  k_cenarion_circle_adj:0 "cénarienne"
- k_exodar:0 "Exodar"
- k_exodar_adj:0 "Exodarienne"
- e_pantheon:0 "Panthéon"
+ k_cenarion_circle_article:0 "le "
+
+ d_cenarion_circle:0 "Cercle cénarien"
+ d_cenarion_circle_adj:0 "cénarienne"
+ d_cenarion_circle_article:0 "le "
+
+ ### Pandaria ###
+ e_pandaria_empire:0 "Pandarie"
+ e_pandaria_empire_adj:0 "pandaroise"
```

Comment:
on ne dirait pas plutôt pandarène ? c'est le cas pour l'adjectif de culture

Reply:
Pour moi c'est l'adjectif de la race donc du coup je me suis dis que j'allais mettre autre chose.

Reply:
comme dit, même une pêche prend l'adjectif pandarène, alors que c'est plutôt lié au territoire je pense plutôt qu'a la culture ou la race https://fr.wowhead.com/item=74660/p%C3%AAche-pandar%C3%A8ne

Reply:
Dans ce cas... Done in cde7fa852